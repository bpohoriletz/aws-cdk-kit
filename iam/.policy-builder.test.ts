import PolicyBuilder from './policy-builder';

describe('new PolicyBuilder()', () => {
  let instance: PolicyBuilder;

  beforeEach(() => {
    instance = new PolicyBuilder();
  });

  test('has defaults set', () => {
    expect(instance).toMatchSnapshot();
  });
});
