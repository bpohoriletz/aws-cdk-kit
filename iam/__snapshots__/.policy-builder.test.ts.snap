// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new PolicyBuilder() has defaults set 1`] = `PolicyBuilder {}`;
